snippets/kotlin/health.code-snippets

```diff
@@ -0,0 +1,100 @@
+{
+  "Initialize health": {
+    "scope": "php",
+    "prefix": "health",
+    "body": [
+      "val health = Health(client)"
+    ],
+    "description": "Create a Health object and initialize it."
+  },
+  "Get HTTP": {
```

Comment:
Could we add a white space between each snippet?

Reply:
Oh yeah, good point. I'll do that.